lib/kb_Msuite/Utils/OutputBuilder.py

```diff
@@ -77,10 +85,13 @@ def build_html_output_for_lineage_wf(self, html_dir, object_name):
         self._write_tabs(html)
 
         # include the single main summary figure
-        html.write('<div id="Plot" class="tabcontent">\n')
-        html.write('<img src="' + plot_name + '" width="90%" />\n')
-        html.write('<br><br><br>\n')
-        html.write('</div>\n')
+        if plot_exists:
+            html.write('<div id="Plot" class="tabcontent">\n')
+            html.write('<img src="' + plot_name + '" width="90%" />\n')
+            html.write('<br><br><br>\n')
+            html.write('</div>\n')
+        else:
+            html.write('<p>The Bin QA Plot was not generated.</p>')
```

Comment:
I think this is maybe a bit too terse. Maybe: "The Bin QA Plot was not generated, this is likely due a result of there being too many bins tested to properly render. Check the log for details"

Reply:
Good call. Unfortunately bin_qa_plot doesn't print its stdout into the logs in the narrative so I'm not sure how the user will see the error message without me tweaking this app some more. I'll make this less terse at any rate.

Reply:
Okay, made the update :heavy_check_mark: